<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataConfigWidget</class>
 <widget class="QWidget" name="DataConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>283</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>283</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>555</width>
    <height>283</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="DCfLayout">
    <item>
     <layout class="QGridLayout" name="ValLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="ValidatorTXT">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ValidatorCBOX">
        <property name="text">
         <string>Needs Validation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Validator">
        <property name="text">
         <string>Validator:</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="GenCmds">
      <property name="text">
       <string>Generation Commands(e.g.: gen &gt;1,multigen &gt;{1,3-4},std &lt;prob.in &gt;1, space sensitive)</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="GenCmdTXT"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
